frontend/templates/views/partials/format-filter.j2

```diff
@@ -0,0 +1,23 @@
+{% do doc.styles.addCssFile('css/components/organisms/format-filter.css') %}
```

Comment:
Hm... Can we find a better name for this? For me format-filter is confusing with the filter dropdown we have on the documentation pages

Reply:
I was thinking about naming it `format-filter-bubble-list` but that seemed a little too long for me? 😄

Reply:
What about `format-filter-bubbles`? 😄At least that's a little shorter.

Reply:
Okay, done 📟